src/applications/income-and-asset-statement/config/chapters/06-royalties-and-other-properties/royaltiesAndOtherPropertyPages.js

```diff
@@ -0,0 +1,231 @@
+import merge from 'lodash/merge';
+import {
+  arrayBuilderItemFirstPageTitleUI,
+  arrayBuilderItemSubsequentPageTitleUI,
+  arrayBuilderYesNoSchema,
+  arrayBuilderYesNoUI,
+  radioUI,
+  radioSchema,
+  textareaUI,
+  textareaSchema,
+  textSchema,
+  yesNoUI,
+  yesNoSchema,
+} from '~/platform/forms-system/src/js/web-component-patterns';
+import currencyUI from 'platform/forms-system/src/js/definitions/currency';
+import { VaTextInputField } from 'platform/forms-system/src/js/web-component-fields';
+import { arrayBuilderPages } from '~/platform/forms-system/src/js/patterns/array-builder';
+import { relationshipLabels, generatedIncomeTypeLabels } from '../../../labels';
+import {
+  otherRecipientRelationshipExplanationRequired,
+  otherGeneratedIncomeTypeExplanationRequired,
+  recipientNameRequired,
+  showRecipientName,
+} from '../../../helpers';
+
+/** @type {ArrayBuilderOptions} */
+const options = {
+  arrayPath: 'royaltiesAndOtherProperties',
+  nounSingular: 'royalty and other property',
```

Comment:
Consider using 'royalty or other property' instead of 'royalty and other property' - the phrases 'cancel adding this royalty and other property' and 'Add another royalty and other property' sound to me like there are two things being added, not one
<img width="406" alt="Screenshot 2024-08-13 at 11 03 07 AM" src="https://github.com/user-attachments/assets/9e1a4340-1fe0-4805-8daa-966bd2c1fc14">

Reply:
I agree that the language for a lot of these sections can be refined. I'd prefer to defer making content changes until we have an official review from the design and CAIA teams for suggested content updates.